scala/lambdas/ingest-parsed-court-document-event-handler/src/main/scala/uk/gov/nationalarchives/ingestparsedcourtdocumenteventhandler/Lambda.scala

```diff
@@ -101,9 +101,11 @@ class Lambda extends LambdaRunner[SQSEvent, Unit, Config, Dependencies] {
             Some(s"attribute_not_exists($assetId)")
           )
         )
+        groupId = s"COURTDOC_$batchRef"
```

Comment:
Could we move these variables below `_ <- logWithFileRef("Written asset to lock table")`